Documentation/Alamofire 5.0 Migration Guide.md

```diff
@@ -23,7 +23,7 @@ Most APIs have changed in Alamofire 5, so this list is not complete. While most
 
 - `SessionManager` has been renamed to `Session` and its APIs have completely changed.
 - `SessionDelegate` has been rebuilt and it’s public API completely changed. The various closure overrides have been removed, with most now able to be replaced with specific Alamofire features. If there is a need for control over something the closures used to provide, feel free to open a feature request.
-- `TaskDelegate` and the various `*TaskDelegate` classes have been removed. All `URLSession*Delegate` handling is now performed by `SessionDelegate`.
+- `TaskDelegate` and the various `TaskDelegate` classes have been removed. All `URLSessionDelegate` handling is now performed by `SessionDelegate`.
```

Comment:
This was intentional, to indicate that all of the various delegate protocols are handled here.

Reply:
Done.Thanks for review.